I"f<p>존재하지 않는 페이지를 찾았을때 404 에러가 나타나는데.
express 기준</p>

<div class="language-javascript highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nx">app</span><span class="p">.</span><span class="nx">use</span><span class="p">(</span><span class="kd">function</span><span class="p">(</span><span class="nx">req</span><span class="p">,</span> <span class="nx">res</span><span class="p">,</span> <span class="nx">next</span><span class="p">){</span>
  <span class="nx">res</span><span class="p">.</span><span class="nx">status</span><span class="p">(</span><span class="mi">404</span><span class="p">).</span><span class="nx">send</span><span class="p">(</span><span class="dl">'</span><span class="s1">Sorry cant find that!</span><span class="dl">'</span><span class="p">);</span>
<span class="p">})</span> <span class="c1">//추가해준다</span>
</code></pre></div></div>
<p>페이지에 에러가 있을 경우(서버에러 500)</p>

<div class="language-javascript highlighter-rouge"><div class="highlight"><pre class="highlight"><code> <span class="nx">next</span><span class="p">(</span><span class="nx">err</span><span class="p">);</span> <span class="c1">//다음 미들웨어를 호출하는 것 에러를 던진다 생각하면 된다.</span>
 
 
 <span class="nx">app</span><span class="p">.</span><span class="nx">use</span><span class="p">(</span><span class="kd">function</span><span class="p">(</span><span class="nx">err</span><span class="p">,</span> <span class="nx">req</span><span class="p">,</span> <span class="nx">res</span><span class="p">,</span> <span class="nx">next</span><span class="p">){</span> <span class="c1">//첫번째는 next에 담긴 err가 넘어오고</span>
  <span class="nx">res</span><span class="p">.</span><span class="nx">status</span><span class="p">(</span><span class="mi">500</span><span class="p">).</span><span class="nx">send</span><span class="p">(</span><span class="dl">'</span><span class="s1">Something broke!</span><span class="dl">'</span><span class="p">);</span>
<span class="p">})</span>
</code></pre></div></div>
:ET